autovalet_lane_detection/utils_legacy/hls_check.py

```diff
@@ -36,6 +36,9 @@ def nothing(x):
 
     # Normal masking algorithm
     #lower_yellow = np.array([15,51, 230])
```

Comment:
get rid of commented lines we're not using

Reply:
still cool